src/system/documents/item.ts

```diff
@@ -375,12 +375,40 @@ export class CosmereItem<
         // Perform the roll
         const roll = await damageRoll(
             foundry.utils.mergeObject(options, {
-                formula: this.system.damage.formula,
+                formula: `${this.system.damage.formula}+${rollData.mod}`,
                 damageType: this.system.damage.type,
                 mod: rollData.mod,
                 data: rollData,
+                source: this.name,
             }),
         );
+        // We want to store the unmodded damage for use in graze calcs
+        // This isn't a particularly perfect solution, but it's functional
+        // only undoing the automatic addition of the selected attribute
+        rollData.baseRoll = (roll.total ?? 0) - (rollData.mod ?? 0);
```

Comment:
Maybe we should pass through a couple of different options, like:
- `@damage.value` - The total damage that was rolled, including mod
- `@damage.unmodded` - The total damage without the modifier
- `@damage.dice` - The total of all the dice (excludes any additions)

I'm not attached to the names, though I do think `baseRoll` isn't super clear.

Reply:
I thought the baseRoll term might be a bit confusing, though I think whatever terminology we use it will require some level of explanation (hence I included it in the hint text). I do quite like the idea of storing it in an object with multiple options though...